skfda/ml/classification/_centroid_classifiers.py

```diff
@@ -113,7 +113,7 @@ def predict(self, X: Input) -> NDArrayInt:
         """
         check_is_fitted(self)
 
-        return self._classes[  # type: ignore[no-any-return]
+        return self.classes_[  # type: ignore[no-any-return]
```

Comment:
Removing the comment results in a mypy error, at least on my end. Should I try to add something like type-checking classes_ right before using it? Because `self.classes_` is only instantiated after calling fit, so while its type is fixed, it might have not been declared yet.

Reply:
Declare the attribute outside of a method, in the class body. That should work fine.

Reply:
I tried declaring the type of classes_, but directly indexing a ndarray results in any type even in that case.
Alternatively, using an auxiliary variable fixes the type, but a variable used only for the return gives a warning.
It might be more readable to be left as is (?).